huaweicloud/services/dc/resource_huaweicloud_dc_hosted_connect.go

```diff
@@ -0,0 +1,447 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product dc
+// ---------------------------------------------------------------
+
+package dc
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceHostedConnect() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceHostedConnectCreate,
+		UpdateContext: resourceHostedConnectUpdate,
+		ReadContext:   resourceHostedConnectRead,
+		DeleteContext: resourceHostedConnectDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(20 * time.Minute),
+			Update: schema.DefaultTimeout(20 * time.Minute),
+			Delete: schema.DefaultTimeout(20 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `The name of the hosted connect.`,
+			},
+			"bandwidth": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				Description: `The bandwidth size of the hosted connect in Mbit/s.`,
+			},
+			"hosting_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `The ID of the operations connection on which the hosted connect is created.`,
+			},
+			"vlan": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				ForceNew:    true,
+				Description: `The VLAN allocated to the hosted connect.`,
+			},
+			"resource_tenant_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `The tenant ID for whom a hosted connect is to be created.`,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
```

Comment:
`Computed` should be deleted for description

Reply:
done